Popular Movies/Popular Movies/View/Cells/DetailsCells/DirectorWriterTableViewCell.h

```diff
@@ -0,0 +1,14 @@
+//
+//  DirectorWriterTableViewCell.h
+//  Popular Movies
+//
+//  Created by Test on 09/11/2017.
+//  Copyright © 2017 Test. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface DirectorWriterTableViewCell : UITableViewCell
```

Comment:
Move constant declaration above class implementation. Follow constant naming convention and use **UpperCamelCase** naming convention.